### PR TITLE
VMPK.VMPK version 0.8.4

### PR DESCRIPTION
**Pull request by Spahatticat**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/25680)

### CODE REVIEW

**Comment by TinyRobby** [comment]
I hope it will work this time!
:)

**Comment by TinyRobby** [comment]
:smile:

**Comment by TinyRobby** [comment]
# 😄

**Comment by TinyRobby** [comment]
what are you doin? 🔢

**Comment by TinyRobby** [comment]
Is there anything I can do?

File: manifests/v/VMPK/VMPK/0.8.4/VMPK.VMPK.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: VMPK.VMPK
+PackageVersion: 0.8.4
+PackageLocale: en-US
+Publisher: VMPK
+PackageName: VMPK
```
Comment (ItzLevvie) [inline manifests/v/VMPK/VMPK/0.8.4/VMPK.VMPK.locale.en-US.yaml:8]:
```suggestion
PackageName: Virtual MIDI Piano Keyboard
```
image

File: manifests/v/VMPK/VMPK/0.8.4/VMPK.VMPK.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.2.0.29
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.0.0.schema.json
+
+PackageIdentifier: VMPK.VMPK
+PackageVersion: 0.8.4
+PackageLocale: en-US
+Publisher: VMPK
+PackageName: VMPK
+License: Copyright (C) 2008-2021 Pedro Lopez-Cabanillas and others
+ShortDescription: Virtual MIDI Piano Keyboard
```
Comment (ItzLevvie) [inline manifests/v/VMPK/VMPK/0.8.4/VMPK.VMPK.locale.en-US.yaml:10]:
```suggestion
ShortDescription: Virtual MIDI Piano Keyboard is a MIDI events generator and receiver.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
For the `Validation-Unapproved-URL` label, an engineer will take a look at it next week.

File: manifests/v/VMPK/VMPK/0.8.4/VMPK.VMPK.locale.en-US.yaml (hunk 2 above)
Comment (TinyRobby) [reply manifests/v/VMPK/VMPK/0.8.4/VMPK.VMPK.locale.en-US.yaml:10]: Thanks a lot! 😄

File: manifests/v/VMPK/VMPK/0.8.4/VMPK.VMPK.locale.en-US.yaml (hunk 1 above)
Comment (TinyRobby) [reply manifests/v/VMPK/VMPK/0.8.4/VMPK.VMPK.locale.en-US.yaml]: 😄

**Comment by TinyRobby** [comment]
📦 will be here soon 😸

**Comment by TinyRobby** [comment]
what do i have to do to fix the URL?

**Comment by TinyRobby** [comment]
My browser crashed for some reason.

**Comment by ItzLevvie** [comment]
> what do i have to do to fix the URL?

An engineer will take a look at this PR in a few days or so.

**Review by ItzLevvie — APPROVED** [review]

**Comment by TinyRobby** [comment]
😄

**Review by Spahatticat — COMMENTED** [review]
😃

**Pull request merged**